api/discovery/aggregated_v2.json

```diff
@@ -364,25 +364,6 @@
             }
           ],
           "version": "v1"
-        },
```

Comment:
I'm really surprised to see a diff in this file just because we're *relaxing* the openapi validation script